main/py-virtualenv/APKBUILD

```diff
@@ -8,21 +9,50 @@ pkgdesc="Virtual Python Environment builder"
 url="http://www.virtualenv.org/en/latest/"
 arch="noarch"
 license="MIT"
-depends="python2 py2-pip"
-makedepends="python2-dev"
+depends=""
+makedepends="python2-dev python3-dev" # add back py-tox for check()
```

Comment:
What does `# add back py-tox for check()` mean? Dependencies for check are defined in `checkdepends`.

Reply:
I figured that out after looking throught the abuild code, could not find any docs providing that info.

I will fix it in a sec.